tla/README.md

```diff
@@ -5,3 +5,5 @@ This directory contains a formal specification of CCF's variant of Raft in TLA+.
 If your Github account supports it, you can also interact with this specification using codespaces:
```

Comment:
Codespaces is now available to everyone: https://github.blog/2022-11-10-whats-new-with-codespaces-from-github-universe-2022/
```suggestion
You can also interact with this specification using codespaces:
```